src/main/java/frc/robot/RobotContainer.java

```diff
@@ -223,7 +226,11 @@ private void configureButtonBindings() {
     }
 
     if (climbSubsystem != null) {
-      buttons2[Constants.startButton].whenPressed(climbEnabling);
+      // enable climb and spool out arms
+      buttons2[Constants.startButton].whenPressed(
+          new SequentialCommandGroup(climbEnabling, climbSequence1));
```

Comment:
Consider a timeout here?

In case there is something wrong with spooling out the arms, so that you can attempt to do something manually?

Reply:
How could I easily add a timeout for this? Would I just do something similar to my increment variable for shooter pressed?

Reply:
https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html?highlight=timeout#withtimeout

I don't know if that works.  but description seems reasonable.

You could also have another button call cancel